OpenDreamShared/Dream/IconAppearance.cs

```diff
@@ -34,11 +34,13 @@ public sealed class IconAppearance : IEquatable<IconAppearance> {
         [ViewVariables] public AppearanceFlags AppearanceFlags = AppearanceFlags.None;
         [ViewVariables] public int Invisibility;
         [ViewVariables] public bool Opacity;
+        [ViewVariables] public bool Override;
         [ViewVariables] public string? RenderSource;
         [ViewVariables] public string? RenderTarget;
         [ViewVariables] public MouseOpacity MouseOpacity = MouseOpacity.PixelOpaque;
         [ViewVariables] public List<uint> Overlays = new();
         [ViewVariables] public List<uint> Underlays = new();
+        [ViewVariables] public List<uint> ClientImages = new(); //this just holds a list of all clientimage appearance ids associated with this one, it is not filtered by client
```

Comment:
As discussed in discord, this is to make the hash of appearances with client images attached unique, since we use the unique appearance of turfs to address them since they don't have entityuids. It's not a great approach, I will investigate alternatives.

Reply:
Removed since we're not using appearances on the turf anymore